src/blockchain/log-processors/profit-loss/update-profit-loss.ts

```diff
@@ -98,6 +104,15 @@ export async function updateProfitLoss(db: Knex, marketId: Address, positionDelt
     newPrice = (oldPrice.multipliedBy(oldPosition.abs())).plus(price.multipliedBy(positionDelta.abs())).dividedBy(position.abs());
   }
 
+  const nextFrozenFunds = getFrozenFundsAfterEventForOneOutcome({
+    frozenFundsBeforeEvent: {
+      frozenFunds: oldFrozenFunds,
+      frozenProfit: oldFrozenProfit,
+    },
+    // TODO Alex - is this right profit from "frozenProfit += available_funds decreased ? realized PL : 0"?
```

Comment:
This isn't right. frozen profit shouldn't be stored or incremented over time. its just part of calculating frozen funds